.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - "./gradlew clean build"
+
+before_deploy: "mv build/libs/ktlint-ruleset-junit-1.0.0.jar build/libs/ktlint-ruleset-junit-$TRAVIS_TAG.jar"
```

Comment:
this has a hardcoded version. In the deploy step, if we specify`file: build/libs/ktlint-ruleset-junit-*.jar` then `mv` is not needed. what do you think?

Reply:
Ah, yes. Ofcourse when we increase the version, it would not work. Will change this.
So, should we get rid of `$TRAVIS_TAG` as well and pick up the version specified in the `build.gradle` file ?

Reply:
yes